tests/strategies/relative_orders_test.py

```diff
@@ -0,0 +1,215 @@
+from dexbot.strategies.relative_orders import Strategy
+from tests.fixtures import fixture_data_RO
+import logging
+import math
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s %(funcName)s %(lineno)d  : %(message)s'
+)
+
+
+class Test_Strategy:
+    def setup_class(self):
+        self.TEST_CONFIG = fixture_data_RO()
+        self.relative_strategy = Strategy(
+            name='worker 1',
+            config=self.TEST_CONFIG)
+
+    def teardown_class(self):
+        pass
+
+    def setup_method(self):
+        pass
+
+    def teardown_method(self):
+        pass
+
+    def test_configure(self):
+        logging.info(self.relative_strategy.configure())
+        account = self.TEST_CONFIG['workers']['worker 1']['account']
+        conditon = ('dexbot-test1' != account)
+
+        if conditon:
+            raise AssertionError('account error!')
+
+    def test_configure_details(self):
+        cf_details = self.relative_strategy.configure_details()
+        logging.info(cf_details)
+        conditon = (cf_details == [])
+        if not conditon:
+            raise AssertionError('Not empty value！')
+
+    def test_error(self):
+        '''Event method return None'''
+        r = self.relative_strategy.error()
+        logging.info(r)
+        if r is not None:
+            raise AssertionError('Event method return None!')
+
+    def test_tick(self):
+        '''Event method return None'''
+        r = self.relative_strategy.tick(
+            '0228fec41a799ec45c218cb441ae249cef1bbcb2')
+        logging.info(r)
+        if r is not None:
+            raise AssertionError('Event method return None!')
```

Comment:
Why you're not using native `assert` statements?

Reply:
Codac reminds me that the assert statement is not working. Can you go through cdacy?

Reply:
This is wrong, I opened #571

Reply:
thanks

Reply:
i will fix it,thank you.